libs/app-shared/src/lib/components/alert-box-item.tsx

```diff
@@ -0,0 +1,100 @@
+import React from "react";
+import { Circle, MoreVert } from "@mui/icons-material";
+import { Stack, Box, Typography, Grid, MenuItem } from "@mui/material";
+import { IconMenuPopup } from '@mf-mui-starter/common-lib';
+import dayjs from 'dayjs';
+
+export interface AlertBoxItemType {
+  title: string,
+  content: string,
+  date: Date;
+  read?: boolean;
+}
+
+export interface AlertBoxItemProps {
+    item: AlertBoxItemType,
+    onClickMenuItem: () => void,
+}
+
```

Comment:
//rafif todo:
//1, add data-testid to the title, date, content and action buttons
//2, add prop of swtich for the display the action buttons(announcement notification no need action buttons)
//3, add prop of redirect url to redirect the user to a new page, as some notification will make the user to some page with forms to submit

Reply:
- What the action button does? is it to redirect user to another page or is there any other function for that button?

Reply:
does the action button used the prop from point , a prop of redirect url to redirect the user to a new page?